package.json

```diff
@@ -79,6 +79,7 @@
     "@percy/playwright": "1.0.7",
     "@playwright/experimental-ct-react": "1.50.1",
     "@playwright/test": "1.50.1",
+    "@storybook/web-components": "^9.0.18",
```

Comment:
This should be defined in `/apps/pie-storybook/package.json`, rather than the base repo `package.json` I think?